presto-cli/src/main/java/com/facebook/presto/cli/ClientOptions.java

```diff
@@ -82,7 +82,10 @@
     public String user = System.getProperty("user.name");
 
     @Option(name = "--password", title = "password", description = "Prompt for password")
-    public boolean password;
+    public boolean showPasswordPrompt;
+
+    @Option(name = "-P", hidden = true)
```

Comment:
David explicitly said he didn't want to have this, I don't think we should add it.